modules/caddyhttp/reverseproxy/reverseproxy.go

```diff
@@ -93,6 +93,7 @@ func (h *Handler) Provision(ctx caddy.Context) error {
 			KeepAlive: &KeepAlive{
 				ProbeInterval:   caddy.Duration(30 * time.Second),
 				IdleConnTimeout: caddy.Duration(2 * time.Minute),
+				MaxIdleConnsPerHost: 32,
```

Comment:
Thanks!

The value is indeed configurable already, so a default of 32 sounds good. We can always lower it in the future if there's a better reason.

Can you please gofmt this file and then I can merge it in!

Reply:
oh, sorry, gofmt && force update PR's branch has been completed